src/pytom_tm/tmjob.py

```diff
@@ -303,7 +305,7 @@ def merge_sub_jobs(self, stats: Optional[list[dict, ...]] = None) -> tuple[npt.N
             return result
 
         if stats is not None:
-            search_space = reduce(lambda x, y: x * y, self.search_size) * self.n_rotations
+            search_space = reduce(lambda x, y: x * y, self.search_size) * - (-self.n_rotations // self.symmetry)
```

Comment:
what is up with the double negative here?
If this is a trick to round up, please use `from math import ceil` and `ceil(self.n_rotations / self.symmetry)`